components/locid_transform/src/provider.rs

```diff
@@ -135,6 +136,28 @@ pub struct AliasesV1<'data> {
     pub subdivision: ZeroMap<'data, UnvalidatedSubdivision, SemivalidatedSubdivision>,
 }
 
+#[icu_provider::data_struct(DirectionalityV1Marker = "locid_transform/directionality@1")]
```

Comment:
nit: markers are often encountered without context, so maybe

```suggestion
#[icu_provider::data_struct(ScriptDirectionV1Marker = "locid_transform/script_dir@1")]
```